database/things/edh.pldb

```diff
@@ -1,15 +1,31 @@
-title edh
+title Đ
 appeared 2019
 type pl
 creators Compl Yue
-website https://github.com/complyue/edh
+description Edh is a dynamically & strongly typed, procedural (thus imperative), value-oriented (i.e. immutable first, yet with non-traditional Object constructs), interpreted programming language, that parasitic on GHC and heavily relying on the Haskell implementation of Software Transactional Memory for unified intuition of concurrency and data-consistency.
```

Comment:
interesting. is the title D or edh?

Reply:
Compl Yue uses both in regular text, but "Đ" in titles. And [Wikipedia says that letter is called "Eth,"](https://en.wikipedia.org/wiki/Eth) also spelt "edh." To be pronounced [/ɛð/](https://en.wikipedia.org/wiki/Help:IPA/English).
Programming Linguistics had me learn Natural Linguistics today :p